cmd/copilot/main.go

```diff
@@ -25,16 +26,23 @@ func init() {
 	cobra.EnableCommandSorting = false // Maintain the order in which we add commands.
 }
 
-func main() {
+func run() int {
+	defer colorable.EnableColorsStdout(nil)()
```

Comment:
I wonder why it wasn't working before 🤔 
We're using the following writers when writing to stdout:
https://github.com/aws/copilot-cli/blob/459f729744c4b3d8e4adad2e832312a2dd1d8e94/internal/pkg/term/log/log.go#L23-L24

which uses your library 🤩 as far as I understand:
https://github.com/fatih/color/blob/a05da93ebe62ca9fc6791d3376ec4dad01196448/color.go#L25-L30

Shouldn't that have been working already or am I misunderstanding how `colorable.NewColorableStdout()` works?

Reply:
Ah, copilot-cli already use go-colorable. Okay, I'll fix in another way soon.